docs/vsphere_ci/scripts/install-openssh.ps1

```diff
@@ -0,0 +1,58 @@
+# Powershell script to configure OpenSSH Server in Windows Server 1809 and later.
+#
+# USAGE
+#    ./install-openssh.ps1 
+#    ./install-openssh.ps1 <path/to/authorized_keys>
+#    ./install-openssh.ps1 -keyfile=<path/to/authorized_keys>
+#
+# OPTIONS
+#    keyfile        Path to `authorized_keys` file (Default: authorized_keys)
+#                   The `authorized_keys` file must contains the public key corresponding to 
+#                   the private key in the `cloud-private-key` secret on the OpenShift cluster.
+#                   See https://github.com/openshift/windows-machine-config-operator#usage
+
+# define param for authorized_keys file path
+param ($keyfile='authorized_keys')
+# validate given keyfile
+if (-not(Test-Path -Path $keyfile -PathType Leaf)) {
+    # log error and stop
+    Write-Error -ErrorAction Stop -Message "Cannot find authorized key file: $keyfile" 
+}
+
+# install OpenSSH server (See: https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse)
+Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
+# create firewall rule for Container Logs
+New-NetFirewallRule -DisplayName "ContainerLogsPort" -LocalPort "10250" -Direction Inbound -Action Allow -Protocol TCP -EdgeTraversalPolicy Allow
```

Comment:
Fair enough. However, I feel introducing a new script should be addressed in a more "atomic" approach using [a separate commit](https://github.com/openshift/windows-machine-config-operator/pull/466/commits/d0365c904e9abd3697eb28d79668e0ada9eb82b7).

Reply:
Point I am making is adding it here in this commit, removing it in another commit is just unnecessary code churn. Moreover given the [discussion](https://github.com/openshift/windows-machine-config-operator/pull/466#discussion_r656479631) as to why it is being added, it might be preferable for it to remain here.